### DISCUSSION TITLE
SEO /test/1234 vs /test-1234 vs /test?id=1234

### DISCUSSION THREAD

**Discussion by VrTech** [vercel/next.js#32968] [Help]
Hi guys! I want a bit of advice for the SEO tips
Can anyone explain the best practice for the crawlable link?
between

`/test/1234`
`/test-1234`
`/test?id=1234`

Which is the best or any better way for SEO points?

**Comment by wwwdepot** [comment]
Use the one that's simple and descriptive. https://developers.google.com/search/docs/advanced/guidelines/url-structure

**Comment by VrTech** [comment]
Thank you.
This really insightful :)
